explorer/data/prelude.carbon

```diff
@@ -4,6 +4,8 @@
 
 package Carbon api;
 
+// Conversion interfaces.
```

Comment:
Should these be more banner-y with a row of all dashes, like:

```
// --------------
// Conversion interfaces.
// ---------------
```

?

Reply:
Sounds good, done.